pyproject.toml

```diff
@@ -2,44 +2,45 @@
 name = "membermanagement"
 version = "0.1.0"
 description = "The Django Application for managing Jacobs University Bremen Alumni Members."
-authors = ["Leonhard Kuboschek <leo@legartis.ai>"]
+authors = ["Tom Wiesing <tkw01536@gmail.com>", "Leonhard Kuboschek <leo@legartis.ai>"]
```

Comment:
This email is outdated, leo@kuboschek.me is more current

Reply:
Ooops forgot about that one

Reply:
Fixed in 18e2393010a10fe0269442d8e15dc66641182b43.